tsconfig.json

```
@@ -22,6 +22,6 @@
       "@/*": ["./src/*"]
     }
   },
```

You can also add the following to change the subtle TypeScript MDX editor hints into red squiggly type errors inside MDX files:

```suggestion
  },
  "mdx": {
    "checkMdx": true
  },
```